python/sidecar/prompts.py

```diff
@@ -1,11 +1,24 @@
-from sidecar.typing import Chunk
+from sidecar.typing import Chunk, TextCompletionRequest
 
 
 def create_prompt_for_summarize(text: str) -> str:
     prompt = f"Please rewrite the following text in a more concise manner:\nTEXT: {text}"
     return prompt
 
 
+def create_prompt_for_text_completion(request: TextCompletionRequest) -> str:
+    prompt = "I am a researcher at a university. You are my research assistant. "
```

Comment:
We might need to change this for other type of documents in the future (not scholar).